packages/composite-checkout-wpcom/src/components/wp-contact-form.js

```diff
@@ -174,135 +171,11 @@ const DomainRegistrationCheckbox = styled.input`
-						}
-						autoComplete="family-name"
-						isError={ lastName.isTouched && ! lastName.isValid }
-						errorMessage={ translate( 'This field is required.' ) }
-					/>
-				</RightColumn>
-			</FieldRow>
-
-			<FormField
-				id={ section + '-email-address' }
-				type="email"
-				label={ translate( 'Email address' ) }
-				placeholder={ translate( 'name@example.com' ) }
-				value={ email.value }
-				onChange={ value =>
-					setContactField( 'email', { value, isTouched: true, isValid: !! value } )
-				}
-				autoComplete="email"
-				isError={ email.isTouched && ! email.isValid }
-				errorMessage={ translate( 'This field is required.' ) }
-			/>
-
-			<FormField
-				id={ section + '-address' }
-				type="text"
-				label={ translate( 'Address' ) }
-				value={ address.value }
-				onChange={ value =>
-					setContactField( 'address', { value, isTouched: true, isValid: !! value } )
-				}
-				autoComplete={ section + ' street-address' }
-				isError={ address.isTouched && ! address.isValid }
-				errorMessage={ translate( 'This field is required.' ) }
-			/>
-
-			<FieldRow>
-				<LeftColumn>
-					<Field
-						id={ section + '-city' }
-						type="text"
-						label={ translate( 'City' ) }
-						value={ city.value }
-						onChange={ value =>
-							setContactField( 'city', { value, isTouched: true, isValid: !! value } )
-						}
-						autoComplete={ section + ' address-level2' }
-						isError={ city.isTouched && ! city.isValid }
-						errorMessage={ translate( 'This field is required.' ) }
-					/>
-				</LeftColumn>
-
-				<RightColumn>
-					<StateSelectWrapper>
-						<StateSelect
-							countryCode={ country.value || 'US' }
-							label={
-								isStateorProvince() === 'state' ? translate( 'State' ) : translate( 'Province' )
-							}
-							name={ section + '-state' }
-							onChange={ event =>
-								setContactField( 'state', {
-									value: event.target.value,
-									isTouched: true,
-									isValid: !! event.target.value,
-								} )
-							}
-							value={ state.value }
-						/>
-					</StateSelectWrapper>
-				</RightColumn>
-			</FieldRow>
-		</React.Fragment>
-	);
-}
-
-AddressFields.propTypes = {
-	section: PropTypes.string.isRequired,
-	contactInfo: PropTypes.object.isRequired,
-	setters: PropTypes.object.isRequired,
-	StateSelect: PropTypes.elementType.isRequired,
-};
-
-function isStateorProvince() {
-	//TODO: Add location detection to return "state" or "province"
-	return 'state';
-}
-
-function PhoneNumberField( {
-	id,
-	isRequired,
-	setContactField,
-	contactInfo,
-	countriesList,
-	PhoneInput,
-} ) {
+function PhoneNumberField( { id, isRequired, contactInfo, countriesList, PhoneInput, setters } ) {
 	const translate = useTranslate();
+	const { updatePhone, updatePhoneNumberCountry } = setters;
 
-	const isError = contactInfo.phoneNumber.isTouched && ! contactInfo.phoneNumber.isValid;
+	const isError = contactInfo.phone.isTouched && ! isValid( contactInfo.phone );
```

Comment:
This is a reminder to myself to make sure that changing this to `phone` doesn't break the payment method submissions.